airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -422,3 +422,55 @@ def create_or_update_glue_job(self) -> str | None:
             self.conn.create_job(**config)
 
         return self.job_name
+
+
+class GlueDataBrewHook(AwsBaseHook):
```

Comment:
I'm not sure should we store it into the `glue.py` or move it into the `databrew.py`. It somehow related to the Glue, but internally in `boto3` it is separate client, and I guess it might have separate AWS API. So in theory both approach could be applicable, that is just question for clarification

cc: @vincbeck @ferruzzi @o-nikolas

Reply:
I wasn't sure which way it should really go, it is shown in the console under Glue. The trigger, however, needs to be named `databrew`, so it probably makes more sense to split it out into its own set of files. I will make the changes, if this is the direction we should go.

Reply:
Yeah, it's in an awkward position and I guess either would work, but I'd lean towards separate in this case as well.

Reply:
There is a simple answer to where to put it.

If I am a user of `databrew` what is the first place that I will look to find databrew related code? (This question is regardless of Airflow itself.. it's the same for doc search or any other thing related to the service)
obviously I will go under `Amazon/aws` but what after?
would it be under `glue`  / `databrew` / `glue_databrew` ?
If most users are going naturally to A then code should be in A.
This is the only question that matters.